snaplet-sqlite-simple.cabal

```diff
@@ -1,5 +1,5 @@
 name:           snaplet-sqlite-simple
-version:        0.4.6
+version:        0.6.0.1
```

Comment:
Initially it was 0.5.0.0 in port 1.0 https://github.com/nurpax/snaplet-sqlite-simple/pull/16/commits/fcc389146d9729f1ed366ba3e6fcfe38c116e0a8, so I assumed I need to bump 1 point up. Would you like to be 0.5.0.0 ?